addons/bus/static/tests/helpers/mock_server.js

```diff
@@ -36,6 +37,13 @@ patch(MockServer.prototype, "bus", {
             const { inactivity_period, im_status_ids_by_model } = data;
             this._mockIrWebsocket__updatePresence(inactivity_period, im_status_ids_by_model);
         }
+        if (event_name === "subscribe") {
+            const sendQueue = this.sendQueue;
+            this.sendQueue = [];
+            for (const values of sendQueue) {
+                this.websocketWorker.broadcast("notification", values);
+            }
+        }
```

Comment:
I think the send queue should not be emptied there. Indeed, we could start another web client which would also require the missed notifications. Moreover, I think we should check the `last_notification_id` parameter in order to match the server behavior as much as possible.